src/main.rs

```diff
@@ -241,98 +247,98 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 
-                      // We're done waiting.
-                      error!("Graceful shutdown timed out. {} active clients being closed", total_clients);
+                        // We're done waiting.
+                        error!("Graceful shutdown timed out. {} active clients being closed", total_clients);
 
-                      let _ = exit_tx.send(()).await;
+                        let _ = exit_tx.send(()).await;
                     });
                 },
 
                 _ = term_signal.recv() => {
                     info!("Got SIGTERM, closing with {} clients active", total_clients);
                     break;
-            },
+                },
 
-            new_client = listener.accept() => {
-                let (socket, addr) = match new_client {
-                    Ok((socket, addr)) => (socket, addr),
-                    Err(err) => {
-                        error!("{:?}", err);
-                        continue;
-                    }
-                };
-
-                let shutdown_rx = shutdown_tx.subscribe();
-                let drain_tx = drain_tx.clone();
-                let client_server_map = client_server_map.clone();
-
-                let tls_certificate = config.general.tls_certificate.clone();
-
-                tokio::task::spawn(async move {
-                    let start = chrono::offset::Utc::now().naive_utc();
-
-                    match client::client_entrypoint(
-                        socket,
-                        client_server_map,
-                        shutdown_rx,
-                        drain_tx,
-                        admin_only,
-                        tls_certificate.clone(),
-                        config.general.log_client_connections,
-                    )
-                    .await
-                    {
-                        Ok(()) => {
-                            let duration = chrono::offset::Utc::now().naive_utc() - start;
-
-                            if config.general.log_client_disconnections {
-                                info!(
-                                    "Client {:?} disconnected, session duration: {}",
-                                    addr,
-                                    format_duration(&duration)
-                                );
-                            } else {
-                                debug!(
-                                    "Client {:?} disconnected, session duration: {}",
-                                    addr,
-                                    format_duration(&duration)
-                                );
-                            }
+                new_client = listener.accept() => {
+                    let (socket, addr) = match new_client {
+                        Ok((socket, addr)) => (socket, addr),
+                        Err(err) => {
+                            error!("{:?}", err);
+                            continue;
                         }
+                    };
```

Comment:
Did you fix indentation here or something else as well? I can't tell from the diff.

Reply:
This was all just formatting, it was indented incorrectly